### ISSUE TITLE
Mint 20: Implement a collapsable system tray icons applet

### ISSUE THREAD

**Issue by fermmm** [linuxmint/cinnamon#9445]
The applet "collapsable system tray" is not longer maintained and stopper working on Mint 20. Can you please pick the code and make it an official Cinnamon applet? or implement that functionality into the system tray applet.
This is a feature that exists in Windows since 1998 and is still present because people needs it to keep the system tray icons organized.

**Comment by icarter09** [comment]
@fermmm you still experiencing this issue?

**Comment by fermmm** [comment]
> @fermmm you still experiencing this issue?

Yes, currently there is no working collapsable applet for the system tray icons. I updated the one I was using "Collapsable systray" and it's still not working, probably abandoned by the developer, it's not compatible with XappStatusIcons
